iroha/src/main.rs

```diff
@@ -11,7 +11,7 @@ const GENESIS: &str = "genesis";
 #[async_std::main]
 #[allow(clippy::non_ascii_literal)]
 async fn main() -> Result<()> {
-    println!("Hyperledgerいろは2にようこそ！");
+    iroha_logger::info!("Hyperledgerいろは2にようこそ！");
```

Comment:
`println` is used here, because logger is usually not initialized by this time. Is it now?

Reply:
Yeah, you right. Will change it!